tests/jballerina-integration-test/src/test/resources/grpc/src/tool/helloWorldGateway.proto

```diff
@@ -0,0 +1,46 @@
+/*
+ *  Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+syntax = "proto3";
+import "google/api/annotations.proto";
+service helloWorldGateway {
+		rpc hello(HelloRequest) returns (HelloResponse) {
+		option (google.api.http) = 
+        		{ post: "/v1/{number}/"
+          		}; 
+		}
+		rpc bye(ByeRequest) returns (ByeResponse) {
+		option (google.api.http) = 
+			{ get: "/v1/" 
+          		};
+		}
```

Comment:
Please check the indentations and formatting.

Reply:
I changed the formatting in the integration test proto files. Please let me know if the formatting is fine or requires further changes